integrations/http/save.go

```diff
@@ -0,0 +1,87 @@
+package http
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"go.uber.org/zap"
+)
+
+const (
+	HeaderContentType = "Content-Type"
+	ContentTypeForm   = "application/x-www-form-urlencoded"
+)
+
+// handleAuth saves a new autokitteh connection with user-submitted data.
+func (h handler) handleAuth(w http.ResponseWriter, r *http.Request) {
+	l := h.logger.With(zap.String("urlPath", r.URL.Path))
+
+	// Check "Content-Type" header.
+	ct := r.Header.Get(HeaderContentType)
+	if ct != ContentTypeForm {
+		l.Warn("Unexpected header value",
```

Comment:
probably better to log unexpected content type than just header value

Reply:
I agree, but because this is happening in all integrations that use a web form to save data, I created a ticket to handle all of them separately from this PR.

https://linear.app/autokitteh/issue/ENG-645/logging-tweaks-based-on-code-review